model/patch_lifecycle.go

```diff
@@ -446,6 +451,21 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 	return patchVersion, nil
 }
 
+func GetParameters(patchDoc *patch.Patch) error {
+	parentPatchId := patchDoc.Triggers.ParentPatch
+	parentPatch, err := patch.FindOneId(parentPatchId)
+	if err != nil {
+		return errors.Wrap(err, "can't get parent patch")
+	}
+	if parentPatch == nil {
+		return errors.Wrap(err, fmt.Sprintf("parent patch '%s' does not exist", parentPatchId))
+	}
+	if downstreamParams := parentPatch.Triggers.DownstreamParameters; downstreamParams != nil {
```

Comment:
Is it necessary to check for `nil`?

Reply:
hmm, I realized checking for nil does nothing, because it's not a pointer. I'll check instead if the length > 0.